yjit.rb

```diff
@@ -18,6 +18,11 @@ def self.stats_enabled?
     Primitive.rb_yjit_stats_enabled_p
   end
 
+  # Check if `--yjit-compilation-log` is used.
+  def self.compilation_log_enabled?
```

Comment:
Don't hate me but, could we simplify all occurrences of `compilation_log` (including `compilation_log.rs` filename) and just rename them to log? We could talk about this as the YJIT log? I don't think we'll want to have multiple separate log streams in YJIT (because that would be inherently repulsive).

Reply:
@maximecb I've gone ahead and renamed everything to simplify to "log". I've isolated all of those changes to [5f81f1b](https://github.com/ruby/ruby/pull/11818/commits/5f81f1b31d7919c1ff9505ead8119c2a591c0ae6) for easy review.

Reply:
Thank you Kevin and also thanks for your patience 🙏